### PR TITLE
Add Fundle install support

### PR DESCRIPTION
**Pull request by SirWrexes**
Hey there,

I'm not a fan of installing plugins straight into where I put my own configs.
Nonetheless, I've been using Fisher for a while until, recently, jorgebucaran/fisher#651 broke everything.
I kept getting annoying warnings telling me to delete my plugin files in order to update them blablabla... but I digress.

I switched to Fundle, which works more elegantly IMO by placing plugins in a dedicated directory tree then sourcing everything.

Luckily, there are several ways of checking if a user has Fundle, and one of them is querying `functions` too see if `__fundle_plugins_dir` exists, which will then serve to tell us where Gitnow has been installed.
I quickly wrote a fix with this method, while retaining the pretty handy variables you created.

Note that `fish_config` could also be set using the value from the standard `__fish_config_dir` variable.

Have a good day ! :wave:

### CODE REVIEW

**Comment by joseluisq** [comment]
> I switched to Fundle, which works more elegantly IMO by placing plugins in a dedicated directory tree then sourcing everything.
> 
> Luckily, there are several ways of checking if a user has Fundle, and one of them is querying `functions` too see if `__fundle_plugins_dir` exists, which will then serve to tell us where Gitnow has been installed.
> I quickly wrote a fix with this method, while retaining the pretty handy variables you created.

IMO your PR looks more like a feature rather than a fix since as I understood you want to add `Fundle` support. Right?
If so please update the title to "Add Fundle install support" or something.

> Note that `fish_config` could also be set using the value from the standard `__fish_config_dir` variable.

Yeah, you could send a new PR introducing that change if you want. `$XDG_CONFIG_HOME` is kind of restricted.

File: conf.d/gitnow_config.fish [hunk 1]
```diff
@@ -8,11 +8,19 @@ set -q fish_functions; or set -g fish_functions "$fish_config/functions"
 set -q fish_completions; or set -g fish_completions "$fish_config/completions"
 set -q GITNOW_CONFIG_FILE; or set -g GITNOW_CONFIG_FILE ~/.gitnow
 
-set -g gitnow_version 2.6.0
+set -g gitnow_version 2.7.0
```
Comment (joseluisq) [inline conf.d/gitnow_config.fish:11]: Why that change? Are you using the latest master? Can you sync your PR branch please?
https://github.com/joseluisq/gitnow/blob/master/conf.d/gitnow_config.fish#L11

**Review by joseluisq — COMMENTED** [review]
It would be also great if you can share the steps to test Gitnow with Fundle.

**Comment by joseluisq** [comment]
And probably you also want to update the `Install` section of `README` with the new install method. https://github.com/joseluisq/gitnow#install

File: conf.d/gitnow_config.fish (hunk 1 above)
Comment (SirWrexes) [reply conf.d/gitnow_config.fish:11]: Oh my bad. I simply pulled from master when forking, then made the `fundle-compat` branch after editting the file.

Comment (SirWrexes) [reply conf.d/gitnow_config.fish:11]: Wait, on master it's 2.6 too.
I honestly don't know what happened there, that is quite odd. I'll revert it.

Comment (joseluisq) [reply conf.d/gitnow_config.fish:11]: Basically you need to sync your fork first.

**Comment by SirWrexes** [comment]
I'm working on adding the other things like editing the README etc.
So, what version number should I keep ? 2.7.0 I assume ?

**Comment by joseluisq** [comment]
> So, what version number should I keep ? 2.7.0 I assume ?

Yes, Just let it as it is as current Gitnow `master`.

**Comment by SirWrexes** [comment]
Alright.

I have added the installation instructions to `README.md`, and squashed that plus the edits to `gitnow_config.fish` into one `feat:` commit.

I also took time to add a disclaimer about 2.7.0 not supporting Fundle yet in the "Latest stable" installation instructions. Since Fundle's README isn't very clear about how to get a particular version of a plugin and it's not extremely user friendly, I thought it was better having the installation method + a note rather than nothing. It will save you some time later too I guess.

Synchronising my fork in between two commits made squashing take longer than expected but everything seems to be in order and working fine.
I tried it out by copying my fork checked out on `fundle-compat` in my Fundle plugin dir and nothing weird came up.

I'll push the squashed commits.

**Comment by SirWrexes** [comment]
Wow seeing the whole history come up in there because of the sync issue sure is a scary thing.
My apologies. I'm kinda new to the PR thing.

**Comment by joseluisq** [comment]
> Wow seeing the whole history come up in there because of the sync issue sure is a scary thing.
> My apologies. I'm kinda new to the PR thing.

Understand, but unfortunately the history is a bit messy I recommend do a new PR with just your commit changes.

**Comment by joseluisq** [comment]
Feel free to sync your fork or re-fork Gitnow again and open a new PR again.

**Comment by SirWrexes** [comment]
I totally understand.
Will do !

**Pull request closed without merging**